streams/src/main/java/org/apache/kafka/streams/state/internals/AbstractRocksDBSegmentedBytesStore.java

```diff
@@ -196,6 +196,15 @@ public void remove(final Bytes key) {
         segment.delete(key);
     }
 
+    @Override
+    public void remove(final Bytes key, final long timestamp) {
```

Comment:
I wasn't sure how to name this method. I initially called `removeRange(key, from, to)`, but I don't want to support a time range with a specific key because time-ordered key schema will delete other keys between `from-key` and `to-key`.

So I thought of just using one timestamp, to make sure this is not called with a time range. But `removeRange(key, timestamp)` does not look like a range. I ended up just calling it `remove`. Any thoughts?